src/index.ts

```diff
@@ -14,19 +14,20 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import * as matrixcs from "./matrix";
-import * as utils from "./utils";
-import request from "request";
+import * as matrixcs from './matrix';
+import * as utils from './utils';
+import request from 'request';
```

Comment:
this feels a bit overly strict. Is it just an IDE thing or a requirement of the rules?

Edit: I'm really hoping it's an IDE thing given a good chunk of the diff is quote changes.

Reply:
It's an IDE thing. The eslint explicitly turns off quote checks.